docs/2.9.0/docs/canton/usermanual/upgrading.rst

```diff
@@ -209,7 +209,35 @@ Version Specific Notes
 Upgrade to Release 2.9
 ^^^^^^^^^^^^^^^^^^^^^^
 
-TODO
+Protocol versions
+"""""""""""""""""
+The recommended protocol versions is 5 (see :ref:`here <protocol_version>` for more information about protocol versions).
+
+Version 2.9 does not offer support for protocol versions 3 and 4.
+If your sync domain is running one of these protocol versions, you need to perform the upgrade described :ref:`below <one_step_migration>`.
```

Comment:
```suggestion
If your synchronizer is running one of these protocol versions, you must upgrade as described :ref:`below <one_step_migration>`.
```

Reply:
On 2.9, it seems we are still using sync domain (and synchronizer will be used in 3.0)